en/micro-integrator/docs/develop/generate-docker-image.md

```diff
@@ -4,52 +4,102 @@ See the topics given below.
-            application with the artifacts created for your ESB project.
-            The name of the ESB project is displayed by default, but it
-            can be changed if required.
-        -   **Application version** : The version of the composite
-            application.
-        -   **Name of the Docker Image** : A name for the Docker image.
-        -   **Docker Image Tag** : A tag for the Docker image to be used
-            for reference. This is optional.
-        -   **Export Destination** : Browse for the preferred location
-            in your machine to export the Docker image.
-
-        -  Once you have entered the required details, click **Next**.
-        -  In the next page, select the ESB projects that you want to
-        include in the Docker image and click **Finish** .  
-        ![Create docker image](../assets/img/create_project/select_artifact_docker.png)  
-        Once the Docker image is successfully created, a message similar
-        to the following appears in your screen.  
-        ![Create docker image](../assets/img/create_project/docker_image_successful.png)
-
-       3. **Generate Docker Image With an Existing Project** : This will use the selected Existing Docker Exporter Project and create a docker image.
+    -  **Generate Docker Image with the Embedded MI**
+
+        1.  Select this option to generate a Docker image with the embedded Micro Integrator runtime of WSO2 Integration studio. 
+
+            !!! Note
+                This is recommended only for testing.
+
+        2.  Click **Next** and enter the following details:
+            
+            ![Create docker image dialog](../assets/img/create_project/generate_docker_image_dialog.png) 
+
+            <table>
+                <tr>
+                    <th>
+                        Parameter
+                    </th>
+                    <th>
+                        Description
+                    </th>
+                </tr>
+                <tr>
+                    <td>
+                        Name of the Application
+                    </td>
+                    <td>
+                        The name of the composite
+                        application with the artifacts created for your ESB project.
+                        The name of the ESB project is displayed by default, but it
+                        can be changed if required.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Application Version
+                    </td>
+                    <td>
+                        The version of the composite
+                        application.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Name of the Docker Image
+                    </td>
+                    <td>
+                        Give a name for the Docker image.
```

Comment:
@rosensilva - could you provide the format for this? Shouldnt it be 'docker-name/repository-name'? Tried but didnt work.

Reply:
you can give a name you prefer for the docker image. @sajithaliyanage please confirm